pkg/scheduler/scheduler.go

```diff
@@ -523,7 +523,7 @@ func (sched *Scheduler) extendersBinding(pod *v1.Pod, node string) (bool, error)
 			continue
 		}
 		return true, extender.Bind(&v1.Binding{
-			ObjectMeta: metav1.ObjectMeta{Namespace: pod.Namespace, Name: pod.Name, UID: pod.UID},
+			ObjectMeta: metav1.ObjectMeta{Namespace: pod.Namespace, Name: pod.Name, UID: pod.UID, HashKey: pod.HashKey},
```

Comment:
missing Tenant field.

Reply:
Tenant field seems added by another commit in this PR.

Reply:
Confirmed